src/formatDate.js

```diff
@@ -8,7 +8,54 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const [oldData, idFromYear, oldYear, fromSymbol] = getFormat(fromFormat);
+  const [newData, , newYear, toSymbol] = getFormat(toFormat);
```

Comment:
Ensure that the `getFormat` function correctly interprets the format strings. If the format strings do not match the expected pattern, this could lead to incorrect parsing of the date.